### ISSUE TITLE
MongoDB engine unknown

### ISSUE THREAD

**Issue by MatthiasHertel** [RocketChat/helm-charts#51]
Hi,

i have a question, why is the MongoDB Engine unknown ?

i used the chart internal mongodb version.

![image](https://user-images.githubusercontent.com/8426566/159486429-68ddcf09-247c-49cd-8ec7-4e6fb6044d4d.png)

Matthias

**Comment by r0zbot** [comment]
I'm not sure, but we just use the Bitnami mongodb helm chart as a dependency. Maybe you could ask there directly?

**Comment by debdutdeb** [comment]
Yup this is something I fixed recently in docker. I'll take a look how to include that in our helm chart

**Comment by debdutdeb** [comment]
@prppedro did you fix this? :eyes:

**Comment by debdutdeb** [comment]
Doesn't look like fixed on master 😬

**Comment by prppedro** [comment]
The fix was adequate, but proven impractical for our Helm implementation, hence we removed the fix.

**Comment by debdutdeb** [comment]
Any discussions you can point me to?

**Comment by r0zbot** [comment]
We've mostly discussed this in DM's, maybe @prppedro  can explain a bit better, but from what I remember the issue with that PR arose when there is more than one helm install on the same namespace, or when trying to upgrade. 

And the problem is a bit trickier to fix as we can't template values to match the deployment-name (and thus make it unique). 

The CI caught this, but only after it was already merged into master too.

**Comment by debdutdeb** [comment]
Fixed in https://github.com/RocketChat/helm-charts/commit/4b6751c9469628ad4f889ff6ca3de9b633ca8b1d
For reference: https://github.com/RocketChat/helm-charts/blob/efed6b2a729b8dba8ab35a92f5fbc37009f825b7/rocketchat/templates/mongodb-init-configmap.yaml#L2-L3
